content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -86,11 +86,8 @@ In the above example, this version is `kubelet.config.k8s.io/v1beta1`.
 
 ## Drop-in directory for kubelet configuration files {#kubelet-conf-d}
 
-As of Kubernetes v1.28.0, the kubelet has been extended to support a drop-in configuration directory. The location of it can be specified with
-`--config-dir` flag, and it defaults to `""`, or disabled, by default.
+In Kubernetes v1.29.0, the drop-in configuration directory feature allows the kubelet to support a drop-in configuration directory, which can be specified using the `--config-dir` flag. By default, it is set to `""`, indicating that the feature is disabled by default. Users need to expliclty opt-in for this feature by setting the `--config-dir` flag to a valid directory path.
 
```

Comment:
```suggestion

For Kubernetes v1.28 to v{{< skew currentVersion >}}, you  can only specify `--config-dir` if you also set
the environment variable `KUBELET_CONFIG_DROPIN_DIR_ALPHA` for the kubelet process (the value
of that variable does not matter).
```

This change is appropriate for a PR targeting _main_; it will render right. Then, once _this_ is merged to main and has been onwards merged to _dev-1.30_, update that branch to mark the feature as beta. You would drop the
`v1.28 to v{{< skew currentVersion >}}` text and code it as (eg) `v1.28 and v1.29`.